lifx_control_panel/utilities/color_thread.py

```diff
@@ -64,6 +70,20 @@ def normalize_rectangles(rects: List[Tuple[int, int, int, int]]):
         for left, top, right, bottom in rects
     ]
 
+cycleColor = (255, 0, 0)
```

Comment:
Should find a way to remove global values from this function.

Reply:
I'm still pretty fresh to python overall - I don't know another method to store a value between calls of the function. if you can suggest some code I'd be happy to test and add it to the PR